### ISSUE TITLE
[Feature Request] Disable Meta data gathering

### ISSUE THREAD

**Issue by abillauer** [ajslater/codex#342]
I know a lot of people use codex for metadata sorting and searching. My primary use case is folder view and at most title search.
Can you add a feature to the Flags screen that disables metadata search. This should also disable the top group options of publisher, Series, Story Arc as well as disable the relevant order by fields.

I imagine this will make polling the libraries much simpler and faster and improve overall performance.

Thanks!

**Comment by bmfrosty** [comment]
I would want this too.

**Comment by ajslater** [comment]
This disables the most useful (to me) features of codex that I've spent 95% of time sweating over. But it was super easy, so I threw it into [v1.5.0](https://github.com/ajslater/codex/releases/tag/v1.5.0) at the last minute.

I made a very sophisticated database browser, filter and searcher and there really is a distinct and vocal contingent of users who mostly just use the file browser I tacked on as a afterthought one day. 🙃
